### PR TITLE
Remove one time fix to user date created

### PR DESCRIPTION
**Pull request by megrogan**

### CODE REVIEW

**Review by hpeebles — APPROVED** [review]

**Comment by megrogan** [comment]
Once this code is merged the user canisters will need to be upgraded ahead of the local_user_index. So don't merge this in until we're ready to upgrade user canisters.

**Review by hpeebles — APPROVED** [review]

**Pull request merged**
